client/my-sites/plan-features-2023-grid/icons/jetpack-ai.tsx

```diff
@@ -0,0 +1,24 @@
+import { SVG, Path } from '@wordpress/primitives';
```

Comment:
**Note:**

This file is a duplicate of  https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/add-ons/icons/jetpack-ai.tsx